src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -601,7 +601,8 @@ inlineCommands = M.fromList $
   , ("thanks", note <$> grouped block)
   , ("footnote", note <$> grouped block)
   , ("verb", doverb)
-  , ("lstinline", skipopts *> dolstinline)
+  , ("lstinline", do options <- option [] keyvals
```

Comment:
To make this cleaner, move `options <- option [] keyvals` down into `dolstinline` and remove the `opts` parameter there.